archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -34,36 +34,35 @@
             <groupId>com.paypal.selion</groupId>
             <artifactId>SeLion</artifactId>
             <version>${selion.version}</version>
+            <scope>test</scope>
```

Comment:
I would vote we keep things simple in the archetype pom file.  Changing to test scope for SeLion is nice to have but not at the cost of making the pom dependencies overly complicated.